gamestonk_terminal/cryptocurrency/binance_model.py

```diff
@@ -0,0 +1,258 @@
+        "15min": client.KLINE_INTERVAL_15MINUTE,
+        "30min": client.KLINE_INTERVAL_30MINUTE,
+        "1month": client.KLINE_INTERVAL_1MONTH,
+    }
+    interval_choices = list(interval_map.keys())
+
+    parser = argparse.ArgumentParser(
+        prog="candle",
+        add_help=False,
+        description="Program to plot candles for binance data",
+    )
+    parser.add_argument(
+        "-i",
+        "--interval",
+        help="Interval to get data",
+        choices=interval_choices,
+        dest="interval",
+        default="1day",
+        type=str,
+    )
+
+    parser.add_argument(
+        "-l", "--limit", dest="limit", default=100, help="Number to get", type=int
+    )
+
+    try:
+        ns_parser = parse_known_args_and_warn(parser, other_args)
+        if not ns_parser:
+            return
+        candles = client.get_klines(
+            symbol=coin,
+            interval=interval_map[ns_parser.interval],
+            limit=ns_parser.limit,
+        )
+
+        # Response
+        # 1499040000000, // Open time
+        # "0.01634790", // Open
+        # "0.80000000", // High
+        # "0.01575800", // Low
+        # "0.01577100", // Close
+        # "148976.11427815", // Volume
+        # 1499644799999, // Close time
+        # "2434.19055334", // Quote asset volume
+        # 308, // Number of trades
+        # "1756.87402397", // Taker buy base asset volume
+        # "28.46694368", // Taker buy quote asset volume
+        # "17928899.62484339" // Ignore.
+
+        candles_df = pd.DataFrame(candles).astype(float).iloc[:, :7]
+
+        candles_df.columns = [
+            "Time0",
+            "Open",
+            "High",
+            "Low",
+            "Close",
+            "Volume",
+            "Time1",
+        ]
+        candles_df.index = pd.to_datetime(
+            (candles_df["Time0"] + candles_df["Time1"]) / 2, unit="ms"
+        )
+
+        plot_candles(
+            candles_df,
+            f"{coin} from {candles_df.index[0].strftime('%Y/%m/%d')} to "
+            f"{candles_df.index[-1].strftime('%Y/%m/%d')}",
+        )
+
+    except Exception as e:
+        print(e, "\n")
+
+
+def balance(coin: str):
+    """
+    Get account holdings for asset
+
+    Parameters
+    ----------
+    coin: str
+        Coin to get holdings of
+
+    Returns
+    -------
+
+    """
+    client = Client(API_BINANCE_KEY, API_BINANCE_SECRET)
+    try:
+        current_balance = client.get_asset_balance(asset=coin)
+        if current_balance is None:
+            print("Check loaded coin")
+            return
+        print("")
+        amounts = [float(current_balance["free"]), float(current_balance["locked"])]
+        total = np.sum(amounts)
+        df = pd.DataFrame(amounts)
+        df.columns = ["Amount"]
+        df.index = ["Free", "Locked"]
+        df["Percent"] = df.div(df.sum(axis=0), axis=1).round(3)
+        print(f"You currently have {total} coins and the breakdown is:")
```

Comment:
You can use this trick: `str(float( number ))`

<img width="253" alt="Captura de ecrã 2021-05-03, às 10 08 33" src="https://user-images.githubusercontent.com/25267873/116859247-94ffdc80-abf7-11eb-880d-8482ec16a9a4.png">